webapp/channels/src/components/drafts/draft_row.tsx

```diff
@@ -37,6 +37,7 @@ import PlaceholderScheduledPostsTitle
 import EditPost from 'components/edit_post';
 
 import Constants, {StoragePrefixes} from 'utils/constants';
+import * as Utils from 'utils/utils';
```

Comment:
why import everything, why not selectively import copyToClipboard

Reply:
Done.